src/main/java/game/applicationBase/Main.java

```diff
@@ -0,0 +1,68 @@
+package game.applicationBase;
+
+import game.Game;
+import game.Player;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.List;
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main (String[] args){
+        // TODO: 06-03-2020 rewrite this mess
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Welcome to our Software Design game.");
+
+        List<File> saveFileList = LocalFileTool.listSaveFiles(Main.class.getClassLoader());
+        System.out.println("Pick an option:");
+        System.out.println("1.\tNew Game");
```

Comment:
After the user has decided to begin a 'New Game' consider showing them the most crucial options such as "quit" to exit the game.